keep/api/routes/providers.py

```diff
@@ -80,7 +82,37 @@ def get_providers(
             "installed_providers": [],
             "is_localhost": is_localhost,
         }
+    
+@router.get(
+    "/export",
+    description="export all installed providers"
+)
+def get_installed_providers(
```

Comment:
Again, you can get all the installed providers today already, why a dedicated endpoint?

Reply:
I wanted all endpoints used for import/export feature to end with /export, for ease of use with scripts and so on (just replace the word in the middle; workflows/providers/etc)
also, I think extra endpoint only for installed providers is much cleaner, as we get a lot of unrelated info with the previous one